examples/src/json-codec/client.rs

```diff
@@ -0,0 +1,23 @@
+use common::{HelloRequest, HelloResponse, JsonCodec};
```

Comment:
We should add a module comment explaining/pointing out that the codegen comes from `build_json_codec_service` in the `build.rs` file.